pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -1,44 +1,53 @@
 package ossm
 
 import (
+	"embed"
 	"fmt"
-	kftypesv3 "github.com/opendatahub-io/opendatahub-operator/apis/apps"
 	configtypes "github.com/opendatahub-io/opendatahub-operator/apis/config"
 	"github.com/opendatahub-io/opendatahub-operator/pkg/kfconfig/ossmplugin"
 	"github.com/opendatahub-io/opendatahub-operator/pkg/secret"
 	"github.com/pkg/errors"
+	"io/fs"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
-	"os"
 	"path"
 	"path/filepath"
 	"strings"
 )
 
+//go:embed templates
+var embeddedFiles embed.FS
+
 type applier func(config *rest.Config, filename string, elems ...configtypes.NameValue) error
 
 func (o *OssmInstaller) applyManifests() error {
 	var apply applier
 
 	for _, m := range o.manifests {
+		targetPath, err := m.targetPath(o.Name, o.Namespace)
```

Comment:
I am wondering if passing one parameter called `rootDirectory` wouldn't be better here. Technically speaking targetPath shouldn't care about name and namespace. It's totally unrelated to where it stores the files.

Reply:
Or maybe even better - we could keep this "templateDir" as a field created for `OssmInstaller` struct (we can combine it with existing outputDir).

Then we don't need any params in the `targetPath` anymore

Reply:
since processTemplate() and targetPath() are both methods on the manifest struct, would it make sense to add it the templateDir field to the manifest struct as well?

Reply:
in fact, maybe it should only be there. good point.

Reply:
done here [c356f9e](https://github.com/maistra/opendatahub-operator/pull/5/commits/c356f9e6b0a784ad078fd0026bc94b39a512140e)